hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/fs/TestFileUtil.java

```diff
@@ -706,15 +706,16 @@ public void testCreateLocalTempFile() throws IOException {
   public void testUnZip() throws IOException {
     // make sa simple zip
     final File simpleZip = new File(del, FILE);
-    OutputStream os = new FileOutputStream(simpleZip); 
-    ZipOutputStream tos = new ZipOutputStream(os);
+    OutputStream os = new FileOutputStream(simpleZip);
+    ZipArchiveOutputStream tos = new ZipArchiveOutputStream(os);
```

Comment:
if you move this to try with resources, the close will be automatic, but I'm not worried about this, as its test code and it keeps the patch smaller

Reply:
good, thouth this is test code, it is also necessary to add this logic.